rosidl_typesupport_cpp/package.xml

```diff
@@ -20,7 +21,6 @@
   <!-- end of group rosidl_typesupport_cpp_packages for bloom -->
 
   <buildtool_export_depend>ament_cmake_core</buildtool_export_depend>
-  <build_export_depend>rosidl_typesupport_c</build_export_depend>
```

Comment:
Since the package exports this dependency (`ament_export_dependencies(rosidl_typesupport_c)` as well as `ament_target_dependencies(${PROJECT_NAME} ... "rosidl_typesupport_c")` this can't be removed.

Reply:
reverted in 5f58fa2